slides/train.md

```diff
@@ -2,3 +2,16 @@
 
 Make them great JavaScript devs and great at working with the framework.
 Have a trainings-pipeline so new devs can be productive quickly.
+
+Note:
+Henning
+
+There are a number of good training resources for your developers. If there aren't any
+on-site classes in your area, there are excellent online classes for free as well as
+paid.
+In most framework communities there are experts in that framework that offer good
+trainings in various locations and online.
+Invest in a dev team library and get them paper or ebooks to share, whichever they
+prefer.
+
+HELP: what do you mean by "trainings-pipeline"
```

Comment:
Do you think it would be helpful to explain what that could look like (the onboarding/trainings-pipeline) or its enough just to mention it (...have this in place ...)